entity-framework/core/saving/concurrency.md

```diff
@@ -3,27 +3,34 @@ title: Handling Concurrency - EF Core
 author: rowanmiller
 ms.author: divega
 
-ms.date: 10/27/2016
+ms.date: 11/27/2017
 
-ms.assetid: bce0539d-b0cd-457d-be71-f7ca16f3baea
 ms.technology: entity-framework-core
 
 uid: core/saving/concurrency
 ---
 # Handling Concurrency
 
-If a property is configured as a concurrency token then EF will check that no other user has modified that value in the database when saving changes to that record.
+If a property is configured as a concurrency token:
 
-> [!TIP]  
-> You can view this article's [sample](https://github.com/aspnet/EntityFramework.Docs/tree/master/samples/core/Saving/Saving/Concurrency/) on GitHub.
+* EF will verify the property value has not been updated after it was fetched.
+* The check occurs when saving changes to that record.
+
+You can view this article's [sample](https://github.com/aspnet/EntityFramework.Docs/tree/master/samples/core/Saving/Saving/Concurrency/) on GitHub.
 
 ## How concurrency handling works in EF Core
 
-For a detailed description of how concurrency handling works in Entity Framework Core, see [Concurrency Tokens](../modeling/concurrency.md).
+For a detailed description of how concurrency handling works in Entity Framework Core, see [Concurrency Tokens](xref:core/modeling/concurrency).
 
 ## Resolving concurrency conflicts
 
-Resolving a concurrency conflict involves using an algorithm to merge the pending changes from the current user with the changes made in the database. The exact approach will vary based on your application, but a common approach is to display the values to the user and have them decide the correct values to be stored in the database.
+
+Resolving a concurrency conflict involves:
```

Comment:
In the original text this sentence worked as a high level overview and the following sentences provided more detail about the steps and the concepts. Now you have two redundant bullet point list. I would suggest having one list of steps first, then calling out the details of what you do in each step as necessary (e.g. the three sets of values that you may need to merge, and the examples of how to merge).

Reply:
I'll work on that.  I should have mentioned my motivation, shorten sentences.